docs/extending-machetli.rst

```diff
@@ -169,7 +168,7 @@ For added convenience, we implement a ``run_successor`` function:
         if len(sys.argv) == 2:
             filename = sys.argv[1]
             try:
-                state = tools.read_state(filename)
+                state = tools.read_state(Path(filename))
```

Comment:
Doesn't the line in the exception also need a Path? If so, I'd do the conversion in line 169 already:
```Python
path = Path(sys.argv[1])
```